app/views/decidim/vocdoni/admin/steps/_vote_stats.html.erb

```diff
@@ -7,7 +7,7 @@
   <div class="card-section table-scroll">
     <ul class="voc-accordion accordion mb-m evote__preview js-votes-count"
         data-wallet="<%= @current_vocdoni_wallet.private_key %>"
-        data-vocdoni-env="<%= Decidim::Vocdoni.config.api_endpoint_env %>"
+        data-vocdoni-env="<%= Decidim::Vocdoni.api_endpoint_env %>"
```

Comment:
As we're changing the callers for `api_endpoint_env`, how about making consistent the callers also of the other accessor (setup_minimum_minutes_before_start)? So we call both of them in the same way (ie `Decidim::Vocdoni.setup_minimum_minutes_before_start`)

This is where it needs to be changed: 

```bash
$ rg setup_minimum_minutes_before_start . | grep Vocdoni.config | cut -f1 -d:
./app/models/decidim/vocdoni/election.rb
./app/forms/decidim/vocdoni/admin/setup_form.rb
./spec/forms/decidim/vocdoni/admin/setup_form_spec.rb

```